# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1491599760 14400
#      Fri Apr 07 17:16:00 2017 -0400
# Node ID 2a6246cc4a6866a1a7f495aff1b2d1ba7bc0b11d
# Parent  b044661132122ad0940029a6d5683069225acfd5
* pt-eval.cc (quit_loop_now): Avoid deprecated variable warnings.

diff --git a/libinterp/parse-tree/pt-eval.cc b/libinterp/parse-tree/pt-eval.cc
--- a/libinterp/parse-tree/pt-eval.cc
+++ b/libinterp/parse-tree/pt-eval.cc
@@ -237,25 +237,25 @@ namespace octave
 // Decide if it's time to quit a for or while loop.
 static inline bool
 quit_loop_now (void)
 {
   octave_quit ();
 
   // Maybe handle 'continue N' someday...
 
-  if (tree_continue_command::continuing)
-    tree_continue_command::continuing--;
+  if (octave::tree_continue_command::continuing)
+    octave::tree_continue_command::continuing--;
 
-  bool quit = (tree_return_command::returning
-               || tree_break_command::breaking
-               || tree_continue_command::continuing);
+  bool quit = (octave::tree_return_command::returning
+               || octave::tree_break_command::breaking
+               || octave::tree_continue_command::continuing);
 
   if (tree_break_command::breaking)
-    tree_break_command::breaking--;
+    octave::tree_break_command::breaking--;
 
   return quit;
 }
 
 namespace octave
 {
   void
   tree_evaluator::visit_simple_for_command (tree_simple_for_command& cmd)
